mloop_multishot.py

```diff
@@ -89,17 +89,22 @@ def verify_globals(config):
     # Get the current runmanager globals
     logger.debug('Getting values of globals from runmanager.')
     rm_globals = rm.get_globals()
-    current_values = [rm_globals[name] for name in config['mloop_params']]
+    current_values = [rm_globals[g.name] for g in config['runmanager_globals']]
 
     # Retrieve the parameter values requested by M-LOOP on this iteration
     logger.debug('Getting requested globals values from lyse.routine_storage.')
-    requested_values = lyse.routine_storage.params
-    requested_dict = dict(zip(config['mloop_params'], requested_values))
+    requested_dict = lyse.routine_storage.params
+    print('requested_dict', requested_dict)
+
+    requested_values = [requested_dict[g.name] for g in config['runmanager_globals']]
+
+    print('requested_values', requested_values)
+    print('current_values', current_values)

```

Comment:
Were these print statements temporary for debugging, or did you want to keep them around? If you want to keep them, let's use `logger` instead of `print()`

Reply:
I think this temporary as it is already gone locally.